### PR TITLE
Create journal-of-the-south-african-veterinary-association.csl

### PR DESCRIPTION
**Pull request by damnation333**
Via https://forums.zotero.org/discussion/51258/style-request-journal-of-the-south-african-veterinary-association
Adapted for journal, book, chapter and Webpage.

### CODE REVIEW

File: journal-of-the-south-african-veterinary-association.csl [hunk 1]
```diff
@@ -0,0 +1,620 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-GB">
+  <info>
+    <title>Journal of the South African Veterinary Association</title>
+    <title-short>JSAVA</title-short>
+    <id>http://www.zotero.org/styles/journal-of-the-south-african-veterinary-association</id>
+    <link href="http://www.zotero.org/styles/journal-of-the-south-african-veterinary-association" rel="self"/>
+    <link rel="template" href="http://www.zotero.org/styles/apa"/>
```
Comment (rmzelle) [inline journal-of-the-south-african-veterinary-association.csl]: Did this roll out of the Visual Editor?

File: journal-of-the-south-african-veterinary-association.csl [hunk 2]
```diff
@@ -0,0 +1,620 @@
...
+    <link href="http://openjournals.net/files/Ref/HARVARD2009%20Reference%20guide.pdf" rel="documentation"/>
```
Comment (rmzelle) [inline journal-of-the-south-african-veterinary-association.csl]: Did you rely on the documentation at this link? That PDF looks to be from 2009.

I looked at http://www.jsava.co.za/index.php/jsava/pages/view/authors, and it has a link to http://aosis.co.za/documents/Harvard_Reference_Style_Guide.pdf (see "The Journal of the South African Veterinary Association uses the Harvard referencing style."), last updated in 2016.

Comment (rmzelle) [inline journal-of-the-south-african-veterinary-association.csl]: AOSIS also has a bunch of journals (http://aosis.co.za/publishing/scholarly-journals), so we could add dependents, but it looks like they use three different styles (http://aosis.co.za/publishing/about-scholarly-journals#manuscript_formatting).

Comment (damnation333) [inline journal-of-the-south-african-veterinary-association.csl]: Let me look into these two things when I have some time over the next couple of days.

File: journal-of-the-south-african-veterinary-association.csl (hunk 1 above)
Comment (damnation333) [inline journal-of-the-south-african-veterinary-association.csl]: Yes. it did.

File: journal-of-the-south-african-veterinary-association.csl (hunk 2 above)
Comment (damnation333) [inline journal-of-the-south-african-veterinary-association.csl]: @rmzelle 
Just gone through it. Not really many changes.
I'll look into checking the AOSIS styles to make dependents.

File: journal-of-the-south-african-veterinary-association.csl [hunk 3]
```diff
@@ -67,8 +67,8 @@
   <macro name="author">
     <names variable="author">
       <name and="symbol" delimiter-precedes-last="never" et-al-min="7" et-al-use-first="6" initialize-with=". " name-as-sort-order="all"/>
-      <et-al font-style="italic"/>
-      <label form="short" prefix=" (" suffix=")" text-case="capitalize-first"/>
+      <et-al font-style="normal"/>
```
Comment (rmzelle) [inline journal-of-the-south-african-veterinary-association.csl]: We can just delete the `font-style="normal"` here, right?

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**